rust/README.md

```diff
@@ -19,14 +19,24 @@ $ cargo run -- --help
 selenium-manager 1.0.0-M1
 Automated driver management for Selenium
 
-Usage: selenium-manager [OPTIONS] --browser <BROWSER>
+Usage: selenium-manager.exe [OPTIONS] --browser <BROWSER>
```

Comment:
Can the `.exe` be printed only when using Windows?
Or maybe just leave it like `Usage: selenium-manager[.exe] [OPTIONS] --browser <BROWSER>`

Reply:
Yes, in fact the `.exe` only appears when running it in Windows. I have just committed a new patch removing it from the README.